src/app/features/reporting/sql-report/sql-report.service.spec.ts

```diff
@@ -262,4 +263,38 @@ describe("SqlReportService", () => {
       },
     ]);
   });
+  it("should convert report into CSV string for version 1 SQL", () => {
+    const mockSqlData: SqlReportRow[] = [
+      {
+        key: "Total students",
```

Comment:
I believe this is the v2 format, not v1?

And maybe indeed we should rename service.getCsvforV2 if that method is only handling that format ...